policyengine_us/variables/gov/states/mt/tax/income/mt_capital_gains_tax_indiv.py

```diff
@@ -0,0 +1,57 @@
+from policyengine_us.model_api import *
+
+
+class mt_capital_gains_tax_indiv(Variable):
+    value_type = float
+    entity = Person
+    label = "Montana net long-term capital gains tax when married couples file separately"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/12/Form_2_2023_Instructions.pdf#page=6"  # Net Long-Term Capital Gains Tax Table
+    defined_for = StateCode.MT
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.mt.tax.income.main.capital_gains
+        # the tax for capital gains comes into effect after 2024
+        if not p.availability:
+            return 0
+        capital_gains = person("long_term_capital_gains", period)
+        taxable_income = person("mt_taxable_income_indiv", period)
+        non_qualified_income = max_(taxable_income - capital_gains, 0)
+
+        filing_status = person.tax_unit(
+            "state_filing_status_if_married_filing_separately_on_same_return",
+            period,
+        )
+        non_qualified_income_gap = (
+            p.threshold[filing_status] - non_qualified_income
```

Comment:
should this be capped at 0? If not add comment why

Reply:
I think not, since we'll use it to judge if the `non_qualified_income ` is over the threshold